README.md

```diff
@@ -16,8 +16,8 @@ This library, ADAL for iOS and macOS, will no longer receive new feature improve
 # Microsoft Azure Active Directory Authentication Library (ADAL) for iOS and macOS
 =====================================
 
-| [Code Samples](https://github.com/azure-samples?utf8=✓&q=active-directory-ios) | [Reference Docs](http://cocoadocs.org/docsets/ADAL/) | [Developer Guide](https://aka.ms/aaddev)
-| --- | --- | --- |
+| [Code Samples](https://github.com/azure-samples?utf8=✓&q=active-directory-ios) | [Reference Docs](http://cocoadocs.org/docsets/ADAL/) | [Developer Guide](https://aka.ms/aaddev) | [Feedback](https://forms.office.com/r/x693ipGeXW)
```

Comment:
The link doesn't seem to work, is this expected? Thanks.

Reply:
Forms is back up. Link should work. Could you try again and let me know.

Reply:
@oldalton Can these changes be merged in?